skimage/exposure/tests/test_exposure.py

```diff
@@ -634,16 +655,33 @@ def test_adjust_gamma_less_one():
 def test_adjust_gamma_greater_one():
```

Comment:
```suggestion
def test_adjust_gamma_exactly_one():
    image = np.arange(0, 256, dtype=np.uint8).reshape((16, 16))
    expected = image.copy()
    result = exposure.adjust_gamma(image, 1.)
    assert_array_equal(result, expected)

def test_adjust_gamma_greater_one():
```

Why not add the test that revealed the bug too?

Reply:
The test that corresponds to the issue is `test_adjust_gamma_one`, line 599 :wink:

Reply:
I can rename `test_adjust_gamma_one` to `test_adjust_gamma_exactly_one` if you wish...

Reply:
I see, you expanded it to include exactly all values exactly once. Previously, the problematic values were getting skipped.

Reply:
Current state is good with me!